src/Cards/Cards.js

```diff
@@ -15,7 +15,7 @@ import './Cards.scss';
           getMovieDetails={props.getMovieDetails}
           retrieveComments={props.retrieveComments}
           ratedMovies={props.ratedMovies.find(ratedMovie => ratedMovie.movie_id === movie.id) || {}}
-          favorites={props.favorites.find(favoritedMovie => favoritedMovie === movie.id) || {}}
+          favorites={props.favorites.find(favoritedMovie => favoritedMovie === movie.id) || ''}
```

Comment:
I think what needs to happen here is .find() needs a .toString() on the end of it. Then we'll pass the PropType in as a string. I don't know why, but when we leave it as a string and say || 0, a 0 will render on the DOM.